src/routes/artist.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+
+const artistController = require('../controllers/artist');
+const albumController = require('../controllers/album');
+
```

Comment:
Extra blank line could be removed here. Having a single line clear between the different sections in a file is fine, in larger files this would get quite cumbersome quite quickly!

This happens in several other files throughout the project